src/messaging/MessageEnvelope.ts

```diff
@@ -155,7 +174,7 @@ export async function encrypt(
     receivedAt: message.receivedAt,
     ciphertext,
     nonce,
-    senderKeyUri: encrypted.keyUri,
-    receiverKeyUri: receiverKey.id,
+    senderKeyUri: encrypted.verificationMethod,
```

Comment:
Should this be the verification method or the key URL?

Reply:
yeah not a fan of the encryption callbacks defined in the sdk alpha. I changed them here to be simply the id of the verification method (the key URL) and made a move to remove the encryption callbacks entirely from the sdk (https://github.com/KILTprotocol/sdk-js/pull/831)

Reply:
Yeah that's exactly why I'm confused about why you're passing the entire verificationMethod as the value instead of just the id.

Reply:
Ahhh now I see you changed the `EncryptCallback` type. But that's pretty confusing since there's already a type called `VerificationMethod` which you are not using for your variable called `verificationMethod`

Reply:
true, we can just stick with keyUri really